cfme/tests/intelligence/reports/test_reports.py

```diff
@@ -461,3 +484,40 @@ def test_reports_online_vms(appliance, setup_provider, provider, request, vm):
 
     view = navigate_to(saved_report, "Details")
     assert vm not in [row.vm_name.text for row in view.table.rows()]
+
+
+@pytest.mark.long_running
+@pytest.mark.customer_scenario
+@pytest.mark.tier(2)
+@pytest.mark.meta(automates=[1725142])
+@pytest.mark.parametrize("file_name", ["test_migration_db_510.backup"], ids=["customer_db"])
+def test_reports_service_unavailable(temp_appliance_preconfig, file_name, restore_db):
+    """
+    Bugzilla:
+        1725142
+
+    Polarion:
+        assignee: pvala
+        casecomponent: Reporting
+        caseimportance: medium
+        initialEstimate: 1/2h
+        setup:
+            1. Load customer database on the appliance.
+        testSteps:
+            1. Navigate to Reports.
+            2. Navigate to `Details` page of the report with >31k rows.
+        expectedResults:
+            1. Reports must be accessible, there should be no 503 service unavailable error.
+            2. Details page must be accessible.
+    """
+    appliance = temp_appliance_preconfig
+    # 1.
```

Comment:
**Suggestion:**
I don't think comments `# 1.` required, while looking into steps, it will be clear what it is doing.

Reply:
I don't really see any harm in keeping those comments as is, I would rather keep them Satya.

Reply:
@valaparthvi Yes, I agree with you, but I never saw such comments in our code base, Also, not sure whether it is documented anywhere :smile: